packages/scss/src/theming/components/_textfield.theme.scss

```diff
@@ -1,10 +1,13 @@
 $textfield: (
 
   font-size: 1rem,
-  padding-top: 1rem,
+  default-palette: (
+    color: _color("text.light")
```

Comment:
is it background color or text color ? If it's for text, call the key "text" rather than color to follow other palette template

Reply:
It's not useful at all. We use a default palette only for material.